source/jormungandr/jormungandr/customredis.py

```diff
@@ -84,6 +97,9 @@ def redis(app, config, args, kwargs):
 
     redis_url = config.get('CACHE_REDIS_URL')
     if redis_url:
-        kwargs['host'] = redis_from_url(redis_url, db=kwargs.pop('db', None))
+        kwargs['write_client'] = redis_from_url(redis_url, db=kwargs.pop('db', None))
+    redis_url_reader = config.get('CACHE_REDIS_HOST_READER')
```

Comment:
One is about `CACHE_REDIS_URL`, and the next one is about `CACHE_REDIS_HOST_READER`. The name are completely different. Is it a naming problem? Did you mixed up between `CACHE_REDIS_URL` and `CACHE_REDIS_HOST`?